### ISSUE TITLE
Faster downloads

### ISSUE THREAD

**Issue by wwood** [leylabmpi/Struo2#34]
Hi,

Thanks for making this tool - seems very helpful. I was interested in downloading the kracken r207 db, and was initially finding it quite slow - wget was giving me 100KB downloads, so would take >1 month to download all the data from your FTP.

This somehow magically got better with 10+ mins of patience so was getting to 1MB/s, but still that would take a lot of time. 

To improve further I used aria2c to establish concurrent connections. So to download the kracken db

```
(mambaforge_install)cl5n007:20221022:~/m/db/struo2/GTDB_release207/aria2c$ wget http://ftp.tue.mpg.de/ebio/projects/struo2/GTDB_release207/kraken2/                           --2022-10-22 06:58:36--  http://ftp.tue.mpg.de/ebio/projects/struo2/GTDB_release207/kraken2/                                                                                  Resolving ftp.tue.mpg.de (ftp.tue.mpg.de)... 192.124.27.90
Connecting to ftp.tue.mpg.de (ftp.tue.mpg.de)|192.124.27.90|:80... connected.                                                                                                 HTTP request sent, awaiting response... 200 OK
Length: 5235 (5.1K) [text/html]                                                                                                                                               Saving to: ‘index.html’
...

(kingfisher-dev)cl5n007:20221022:~/m/db/struo2/GTDB_release207/aria2c$ aria2c -c -x 8 `grep href index.html |grep -v Parent |grep -v Name |sed 's/.*href=.//; s/".*//' |sed 's=^=http://ftp.tue.mpg.de/ebio/projects/struo2/GTDB_release207/kraken2/='`
...
[#8116ba 790MiB/107GiB(0%) CN:5 DL:6.6MiB ETA:4h33m33s]
```
i.e. faster.

But then I see I'm getting some 416 errors for some files. It would be helpful just to have a single tar file for download instead of having to get each file individually I suppose. Sorry getting a bit rambling here - HTH. I know it seems pretty trivial but maybe instructions for downloading from FTP in the readme would be helpful.

**Comment by nick-youngblut** [comment]
You can use https://github.com/leylabmpi/Struo2/blob/master/util_scripts/database_download.py to help with downloading all of the files

**Comment by wwood** [comment]
FWIW, removing the `-c` fixed those errors, but I also had to run them through aria2c individually, using GNU parallel.

Hadn't seen that util sorry - not very familiar with those underlying python libs but it wasn't obvious to me from the README if it would speed it up. Also, it seems the link to that script in the README table of contents is broken.

Anyway, I got what I needed - feel free to close - was just attempting to be helpful.
ben
